src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -96,6 +96,50 @@ internal IOperation GetOperationInternal(SyntaxNode node, CancellationToken canc
 
         protected abstract IOperation GetOperationCore(SyntaxNode node, CancellationToken cancellationToken);
 
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
i suppressed this public API warning as i don't think it applies to these members.  As the members do not share a prefix set of parameters, there should be no compat issues with having both of them have optional members.

Note that the equivalent APIs already shipped at the C#/VB-SemanticModel level without problems.  So this is just bringing that pattern up to the base SemanticModel type.